### ISSUE TITLE
Have a look at qr stream

### ISSUE THREAD

**Issue by Septolum** [mofosyne/tagdrop#1]
https://github.com/xloem/qrstream

Seems rather similar, might have some useful code/ideas

**Comment by mofosyne** [comment]
yup noted, will certainly be happy to integrate concepts together
